### PR TITLE
SC-4037 Landing Page "Registrieren" Link + New Registrieren Page

### PR DESCRIPTION
**Pull request by pkleybolte**
# Description
 WIP. The accordion should get styled, but the SCSS file is currently not loaded.

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-4037

## Changes
Remove long text from registration page and replace it by an accordion addressing the different categories of schools.

## Screenshots of UI changes

Screen Shot 2020-05-07 at 16 42 20

Screen Shot 2020-05-07 at 16 42 08

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: views/community/community.hbs [hunk 1]
```diff
@@ -1,15 +1,113 @@
 <section class="section-community">
+	
 	{{#embed "authentication/section" class="content-width-max"}}
 	{{#content "body"}}
 
 		{{#unless loggedin}}
 			<h2 class="section-title center">{{title}}</h2>
```
Comment (dominikbruechner) [inline views/community/community.hbs:7]:
```suggestion
			<h1 class="h2 section-title center">{{title}}</h2>
```

File: views/community/community.hbs [hunk 2]
```diff
@@ -1,15 +1,113 @@
...
 		{{/unless}}
-		<h3 class="h4">Jetzt für die HPI Schul-Cloud registrieren!</h3>
 
-		<img src="/images/community.jpg" role="presentation" class="mb-2 w-100"
+		<img src="/images/community.svg" role="presentation" class="mb-2 w-100"
 			style="height: auto; border-radius: 4px;" width="1000" height="667" loading="lazy">
-		<p>
+
+		<h3 class="h5 mb-2">Wählen Sie Ihr Bundesland oder das Projekt dem Sie angehören.</h3>
```
Comment (dominikbruechner) [inline views/community/community.hbs:13]:
```suggestion
		<h2 class="h5 mb-2">Wählen Sie Ihr Bundesland oder das Projekt dem Sie angehören.</h3>
```

File: controllers/community.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ router.get('/', (req, res, next) => {
 		}
 		return Promise.resolve(template);
 	}).then(template => res.render(template, {
-		title: 'Mitmachen',
+		title: 'Jetzt mitmachen!',
```
Comment (CeEv) [inline controllers/community.js:16]: Can you please add this to the localization en.json and de.json?

Comment (dominikbruechner) [reply controllers/community.js]: > @dominikbruechner n21-schoolcloud-logo.svg and th-schoolcloud-logo.svg is theming content, is the location well or must it changed?

I think its fine, as it is just displayed on the "Registrieren" Overview Page

File: views/community/community.hbs [hunk 4]
```diff
@@ -1,15 +1,113 @@
...
 
 		{{#unless loggedin}}
-			<h2 class="section-title center">{{title}}</h2>
+			<h1 class="h2 section-title center">{{$t "community.title"}}</h1>
 		{{/unless}}
-		<h3 class="h4">Jetzt für die HPI Schul-Cloud registrieren!</h3>
 
-		<img src="/images/community.jpg" role="presentation" class="mb-2 w-100"
+		<img src="/images/community.svg" role="presentation" class="mb-2 w-100"
 			style="height: auto; border-radius: 4px;" width="1000" height="667" loading="lazy">
```
Comment (dominikbruechner) [inline views/community/community.hbs:11]:
```suggestion
			style="height: auto; border-radius: 4px; opacity: 0.8;" width="1000" height="667" loading="lazy"/>
```

File: views/community/community.hbs [hunk 5]
```diff
@@ -1,15 +1,113 @@
...
+			</div>
+			<div id="collapseMint" class="collapse" aria-labelledby="mintAccordion" data-parent="#accordionRegistration">
+			<div class="card-body p-1">
+				Die Anmeldung für Schulen aus dem nationale Excellence-Schulnetzwerk MINT-EC erfolgt über folgenden Link:<br>
+				<a href="https://www.mint-ec.de/veranstaltungen/849-hpi-schul-cloud-teilnahme-an-der-zweiten-pilotphase/">https://www.mint-ec.de/veranstaltungen/849-hpi-schul-cloud-teilnahme-an-der-zweiten-pilotphase/</a><br><br>
+
+				Für Fragen wenden Sie sich bitte an <a href="mailto:schul-cloud@mint-ec.de">schul-cloud@mint-ec.de</a>. Vielen Dank!
+			</div>
+			</div>
+		</div>
```
Comment (dominikbruechner) [inline views/community/community.hbs:91]: ist mit Florian Sittig und ChM abgestimmt, diese Karte kann raus
```suggestion

```

Comment (dominikbruechner) [reply views/community/community.hbs:91]: nicht die svgs dazu vergessen, falls neue dazukamen ;)

**Pull request merged**